apps/shelter-web/src/app/shared/components/map/map.tsx

```diff
@@ -113,6 +120,20 @@ export function Map(props: TMap) {
           key={marker.id}
           position={toGoogleLatLng(marker.position)}
           zIndex={99}
+          onClick={() =>
```

Comment:
we'd probably want the `onClick` event passed in as a prop through `markers`, as map should be pretty generic